packages/client/package.json

```diff
@@ -10,45 +10,35 @@
     "start": "cross-env BROWSER=none react-scripts start",
     "prebuild": "cd ../common && yarn build",
     "build": "react-scripts build",
+    "lint": "eslint src/ --ext .ts,.tsx",
+    "prepare": "cd ../.. && yarn prepare",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "codegen": "graphql-codegen --config codegen.yml && scripts/patch-generated-graphql",
-    "codegen:watch": "watch 'yarn codegen' ../main/schema src/documents",
-    "postinstall": "cd ../.. && yarn postinstall"
+    "codegen:watch": "watch 'yarn codegen' ../main/schema src/documents"
   },
   "eslintConfig": {
     "extends": "react-app"
   },
   "browserslist": [
-    "electron >= 4"
+    "electron >= 6"
   ],
   "dependencies": {
     "@material-ui/core": "^4.0.0",
     "@material-ui/icons": "^4.0.0",
     "@reach/router": "^1.2.1",
-    "@types/autosuggest-highlight": "^3.1.0",
-    "@types/graphql": "^14.2.2",
-    "@types/is-url": "^1.2.28",
-    "@types/marked": "^0.6.5",
-    "@types/node": "11.11.3",
-    "@types/object.pick": "^1.3.0",
-    "@types/reach__router": "^1.2.4",
-    "@types/react": "^16.8.22",
-    "@types/react-autosuggest": "^9.3.8",
-    "@types/react-dom": "^16.8.4",
-    "@types/string-hash": "^1.1.1",
     "apollo-cache-inmemory": "^1.6.2",
     "apollo-client": "^2.6.2",
     "autosuggest-highlight": "^3.1.1",
     "clsx": "^1.0.4",
-    "electron": "^4.1.0",
     "email-addresses": "^3.0.3",
     "graphql": "^14.4.2",
     "graphql-tag": "^2.10.1",
     "graphql-transport-electron": "^1.0.1",
-    "immutable": "^4.0.0-rc.12",
+    "immutable": "^3.8.2",
     "is-url": "^1.2.4",
     "marked": "^0.6.2",
+    "moment": "^2.24.0",
```

Comment:
Now that dependencies are not hoisted anymore there were errors importing packages that were listed as dependencies in one of the other packages. Client depends on `moment`, and now that needs to be made explicit by listing the dependency in `package.json`.